rocky/rocky/views/mixins.py

```diff
@@ -380,21 +379,17 @@ def get_children_input_oois(children_reports: list[Report]) -> set[str]:
         return {input_ooi for child_report in children_reports for input_ooi in child_report.input_oois}
 
     @staticmethod
-    def report_type_summary(reports: list[Report]) -> dict[str, int]:
+    def report_type_summary(reports: list[Report]) -> dict[str, int] | None:
```

Comment:
I don't understand why None is added here, the method can never return None?

Reply:
Some report types does not have a summary, I did a print of the log when accessing the summary:
<img width="1056" alt="image" src="https://github.com/minvws/nl-kat-coordination/assets/30832687/c21b4388-5567-485c-8034-21cc448017fc">

Reply:
Normally presentation logic like this should be done in the view / presentation code and not in data / query code like this. But we want to release, so we will keep this for now.